Source/items.cpp

```diff
@@ -4707,6 +4707,33 @@ std::string DebugSpawnUniqueItem(std::string itemName)
 
 	return StrCat(cmdLabel, "Item generated successfully - iterations: ", i);
 }
+
+std::string DebugCreateItem(uint32_t seed, uint8_t idx, uint16_t iCreateInfo, uint32_t dwBuff, uint16_t value)
+{
+	std::string cmdLabel = "[createitem] ";
+	Item createdItem = {};
+
+	RecreateItem(*MyPlayer, createdItem, static_cast<_item_indexes>(idx), iCreateInfo, seed, value, (dwBuff & CF_HELLFIRE) != 0);

```

Comment:
Could also put it here. Might look better.
```suggestion
	RecreateItem(*MyPlayer, createdItem, static_cast<_item_indexes>(idx), iCreateInfo, seed, value, (dwBuff & CF_HELLFIRE) != 0);
	SetupItem(createdItem);
```